bundles/org.openhab.binding.growatt/src/main/java/org/openhab/binding/growatt/internal/GrowattChannels.java

```diff
@@ -186,7 +186,27 @@ public UoM(Unit<?> units, float divisor) {
             // reactive 'power' resp. 'energy'
             new AbstractMap.SimpleEntry<String, UoM>("rac", new UoM(Units.VAR, 10)),
             new AbstractMap.SimpleEntry<String, UoM>("erac-today", new UoM(Units.KILOVAR_HOUR, 10)),
-            new AbstractMap.SimpleEntry<String, UoM>("erac-total", new UoM(Units.KILOVAR_HOUR, 10))
+            new AbstractMap.SimpleEntry<String, UoM>("erac-total", new UoM(Units.KILOVAR_HOUR, 10)),
+
+            /*
+             * ============== CHANNELS ADDED IN PR #17795 ==============
+             */
```

Comment:
Is this really needed, as this is already part of Git history?

Reply:
For me as the maintainer it is really important. The thing has about 100 possible channels based on perhaps 150 possible json field name aliases, and it is incredibly difficult to keep track in my mind what field names map to what channels on what devices and to have an audit trail in the code. I am already working on the next PR too which adds another possible 20 channels for 2 other devices, and I will follow on from the same commenting nomenclature with that one.